### ISSUE TITLE
Incorrect Timezone Label

### ISSUE THREAD

**Issue by adampurnell** [ludumdare/ludumdare#1406]
This may be related to #1383 but timezone is displayed as BST (British Summer Time) instead of GMT.
![incorrecttimezone](https://user-images.githubusercontent.com/6763862/33490795-25811a92-d6b0-11e7-91e5-dd042e64b0eb.png)

The actual time is correct and the alt-text displays the correct timezone.

**Comment by w96k** [comment]
I have a similar issue, but my time is incorrect. See screenshot below

<img width="400" alt="image" src="https://user-images.githubusercontent.com/22467404/33500490-01ad5754-d6ea-11e7-9229-0d45db0b6541.png">

**Comment by povrazor** [comment]
#589

**Comment by adampurnell** [comment]
I live in the United Kingdom.
![incorrect2](https://user-images.githubusercontent.com/6763862/33507703-286f82dc-d6ee-11e7-9702-276f1e958c05.png)

**Comment by hsandt** [comment]
Since this issue has been filed before #1411 and is specifically about Summer time switch, unlike #589 which is very generic and will probably not be fixed before a long time, I'd suggest reopening this ticket, with a description like "Timezone BST used in the UK even in Winter time", so we can focus on the Summer time switch issue and fix it (it's just about splitting a big task in smaller tasks with a clear definition of when it is solved).

To quote @mikekasprzak in #589 :

> BST should be fixed, but I expect it to be wrong again in December ('cause I don't know it's going to think your timezone is).

Basically, the timezone was switched from GMT to BST for the whole year (in the UK) at some point, which reversed the problem. No commit was specified, though, so not sure how it was done.

There may be the same problem with IST in Ireland.
